frontend/src/user/components/UserSettingsForm.tsx

```diff
@@ -0,0 +1,18 @@
+import { Divider, RedButton } from "src/components";
+import { UserProps } from "src/user/types";
+import { ChangePasswordForm, EditCredentialsForm } from "src/user/components";
+
+const UserSettingsForm = ({ user }: UserProps) => {
+  return (
+    <div className="flex flex-col">
+      <Divider label="Basic Information" />
+      <EditCredentialsForm user={user} />
+      <Divider label="Password" />
+      <ChangePasswordForm />
+      <Divider label="Delete Account" />
+      <RedButton type="submit">Delete Account</RedButton>
+    </div>
```

Comment:
realised that alot of the spacing is done within the components itself using `mb`, can we use the `gap` utility class of this parent flex box to do the spacing? alot easier to manage since it's done within the single parent component